apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.scss

```diff
@@ -0,0 +1,10 @@
+@import "~@angular/cdk/overlay-prebuilt.css";
+@import '~fundamental-styles/dist/menu';
```

Comment:
I think it is not necessary
It is already defined [here](https://github.com/SAP/fundamental-ngx/blob/90a091104c2e87e50f46117ffd5963072351e068/apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.scss)